src/components/Materialize/Dropdown.vue

```diff
@@ -55,7 +55,7 @@
 
         let parsed = this.id + this._uid
 
-        return parsed.replace(/[!"#$%&'()*_+,./:;<=>?@[\]^`{|}~ ]/g, '\\$&')
+        return parsed.replace(/[!"#$%&'()*+,./:;<=>?@[\]^`{|}~ ]/g, '\\$&')
```

Comment:
Collections can also be created with the following characters in their names: `!$%&()*+./:;<=>?@[]^`{}|~` and spaces:

![screenshot from 2017-09-28 11-24-31](https://user-images.githubusercontent.com/12997967/30959163-a100ab18-a43f-11e7-87fd-3a67e19bed59.png)